python/dataproc_templates/jdbc/README.md

```diff
@@ -0,0 +1,149 @@
+# 1. JDBC To JDBC
+
+Template for reading data from JDBC table and writing them to a JDBC table. It supports reading partition tabels and write into partitioned or non-partitioned tables.
+
+## Arguments
+
+* `jdbctojdbc.input.url`: JDBC input URL
+* `jdbctojdbc.input.driver`: JDBC input driver name
+* `jdbctojdbc.input.table`: JDBC input table name
+* `jdbctojdbc.input.partitioncolumn`: JDBC input table partition column name
```

Comment:
specify the optional arguments (like partitioncolumn, lower bound etc) in readme